src/workspaces/implementations/npm.ts

```diff
@@ -12,6 +12,6 @@ export function getNpmWorkspaceRoot(cwd: string): string {
 }
 
 export function getNpmWorkspaces(cwd: string): WorkspaceInfo {
-  const yarnWorkspacesRoot = getNpmWorkspaceRoot(cwd);
-  return getWorkspaceInfoFromWorkspaceRoot(yarnWorkspacesRoot);
+  const npmWorkspacesRoot = getNpmWorkspaceRoot(cwd);
```

Comment:
It's odd that this was previously using a yarn workspace in a file with npm in the name, but it would be good to understand why that was done and add extra cases if appropriate instead of just getting rid of it. @kenotron do you know more about the background here?

Reply:
Since it is just the name of the constant, I think it might have been a typo. I am guessing this code might have been copied from the identical `src/workspaces/implementations/yarn.ts` file. I might be wrong though!

Reply:
copy pasta :P